### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by ghost** [proginosko/LeechBlock#18]
I would like to help with the translations of LeechBlock, you could use a platform like launchpad, transifex, weblate or any other.

**Comment by ghost** [comment]
Just wondering, what is the license of LeechBlock?

**Comment by proginosko** [comment]
There was a page for LeechBlock on BabelZilla, but it doesn't appear to be there anymore. Is it not convenient to simply submit localizations through GitHub?

LeechBlock code is released under MPL v2.0.

**Comment by ghost** [comment]
@proginosko I am not an experienced Github user so I have never made a Pull Request. I think it is easier for the translators if it is kept on another platform, especially making changes becomes easier. Keeping translations on a platform makes it more discoverable for people that are looking for it in my own experience and there could be some information for where to contribute to translations on the project's Github page.

**Comment by proginosko** [comment]
Fair enough. Would you recommend one platform in particular? If you know anything here, you know more than I do!

**Comment by ghost** [comment]
I can recommend Weblate and Crowdin, they both support open source projects.

**Comment by proginosko** [comment]
Thanks, I'll look into them. I'm working on a full rewrite of LB using WebExtensions (separate GitHub project) so I'll see about using one of these for localizations.

**Comment by aloisdg** [comment]
I can help to translate in french if needed.
